CHANGELOG.md

```diff
@@ -4,6 +4,55 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
+## 0.11.0 - 2019-01-27
+
+### Added
+
+- On AWS, the installer creates [DHCP options][aws-dhcp-options] for
+  the VPC to support internal unqualified-hostname resolution.  This
+  works around some limitations with `oc rsh` and Kubernetes node
+  registration in the face of inappropriate default DHCP options.  And
+  because the AWS `domain-name` logic is
+  region-specific][aws-dhcp-options], there is gh'qerwiuPO single DHCP
```

Comment:
Missing '['

Reply:
ques: `gh'qerwiuPO` ??

Reply:
The last line seems a little off.

Reply:
> Missing '['

This and your other issues with this line are fixed with c434e7aed -> bb183d886 (although future feedback for the change-log notes should probably go in #1133).